generator/generators/RustGenerator.py

```diff
@@ -0,0 +1,74 @@
+'''
+ Copyright (c) 2018 HERE Europe B.V.
+ See the LICENSE file in the root of this project for license details.
+'''
+
+from generator.tree.nodes.resources import (Vector, Multivector, Instance, RawData, BoundResource,
+    Archive as ArchiveResource)
+from generator.tree.nodes.trivial import Structure, Constant
+from generator.tree.nodes.archive import Archive
+from .BaseGenerator import BaseGenerator
+
+import re
+
+class RustGenerator(BaseGenerator):
+
+    RESERVED_KEYWORDS = [
```

Comment:
Nice, we might want to add this for other languages as well :-)

Reply:
It was quite easy to find a list of keywords for Rust. Also not that hard to find them for C++: http://en.cppreference.com/w/cpp/keyword. I will do same for C++.